mmv1/third_party/terraform/resources/resource_logging_project_sink.go

```diff
@@ -55,13 +62,36 @@ func resourceLoggingProjectSinkCreate(d *schema.ResourceData, meta interface{})
 	id, sink := expandResourceLoggingSink(d, "projects", project)
 	uniqueWriterIdentity := d.Get("unique_writer_identity").(bool)
 
-	_, err = config.NewLoggingClient(userAgent).Projects.Sinks.Create(id.parent(), sink).UniqueWriterIdentity(uniqueWriterIdentity).Do()
-	if err != nil {
-		return err
+	create := true
+	name := d.Get("name").(string)
+	for _, reserved := range ReservedLoggingProjectSinks {
+		if name == reserved {
+			if err := resourceLoggingProjectSinkRead(d, meta); err != nil {
```

Comment:
I'm not sure I understand how this part works, because I don't think an id has been set yet. Won't this Read always return `nil`?